src/main/java/com/training/web/pages/flipkart/ProductPage.java

```diff
@@ -0,0 +1,53 @@
+package com.training.web.pages.flipkart;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class ProductPage {
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    private WebDriver driver;
+    private WebDriverWait wait;
+    By name = By.className("B_NuCI");
+    By price = By.xpath("//*[@id=\"container\"]/div/div[3]/div[1]/div[2]/div[2]/div/div[3]/div[1]/div/div[1]");
+
+    public ProductPage(WebDriver driver) {
+        this.wait = new WebDriverWait(driver, 30);
+        this.driver = driver;
+    }
+
+    public ProductPage clickSize() throws InterruptedException {
+        Thread.sleep(1000);
+        driver.findElement(By.xpath("//*[text()='6']")).click();
+        return this;
+    }
+
+    public ProductPage addToCart() throws InterruptedException {
+        Thread.sleep(1000);
+        driver.findElement(By.xpath("//*[@id=\"container\"]/div/div[3]/div[1]/div[1]/div[2]/div/ul/li[1]/button")).click();
```

Comment:
flaky xpath - use a more optmised locator/xpath. Also this should not be hardcoded inside the function.